api/MediaStreamTrack.json

```diff
@@ -560,10 +560,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
I think the entire `isolated` feature here should be removed from your data — because as far as I can from https://w3c.github.io/mediacapture-main/ the MediaStreamTrack interface has no `isolated` member.

Reply:
This is from https://w3c.github.io/webrtc-identity/identity.html#isolated-track.